locale/de.po

```diff
@@ -516,27 +587,49 @@ msgid "_Advanced"
 msgstr "_Erweitert"
 
 #: src/paperwork/frontend/mainwindow/appmenu.xml.h:4
+msgid "_Diagnostic"
+msgstr "_Diagnose"
+
+#: src/paperwork/frontend/mainwindow/appmenu.xml.h:5
 msgid "Optimize index"
 msgstr "Index optimieren"
 
-#: src/paperwork/frontend/mainwindow/appmenu.xml.h:5
+#: src/paperwork/frontend/mainwindow/appmenu.xml.h:6
 msgid "Reindex all the documents"
 msgstr "Alle Dokumente erneut indizieren"
 
-#: src/paperwork/frontend/mainwindow/appmenu.xml.h:6
+#: src/paperwork/frontend/mainwindow/appmenu.xml.h:7
 msgid "Redo OCR on all the documents"
 msgstr "OCR für alle Dokumente wiederholen"
 
-#: src/paperwork/frontend/mainwindow/appmenu.xml.h:7
+#: src/paperwork/frontend/mainwindow/appmenu.xml.h:8
 msgid "_About"
 msgstr "_Über"
 
-#: src/paperwork/frontend/mainwindow/appmenu.xml.h:8
+#: src/paperwork/frontend/mainwindow/appmenu.xml.h:9
 msgid "_Quit"
 msgstr "_Beenden"
 
-#~ msgid "Invalid date"
-#~ msgstr "Ungültiges Datum"
+#~ msgid "Image rotation of %d degrees"
```

Comment:
Just FIY, you can remove these messages. They are messages that were previously translated but are not found in Paperwork's code anymore